File: src/SOP8/Base/Base.Model/Sop/Component/Comment.cs
using System;
using dnsDapperDBUtil.DataAccessLayer.IDAL;

namespace Base.Model.Sop.Component
{
	public class Comment : Table
	{
		public enum Fields { compn_sn, contents };
		public enum WriteFields { compn_sn, contents };

		public int compn_sn { get; set; }
		public string contents { get; set; }

		public static string TableName { get { return "so_compn_cm"; } }

		public override string GetTableName()
		{
			return TableName;
		}

		public override string GetPrimaryCondition()
		{
			return string.Format("{0} = {1}", Fields.compn_sn, compn_sn);
		}

		public override Type GetFieldType()
		{
			return typeof(Fields);
		}

		public override Type GetWriteFieldType()
		{
			return typeof(WriteFields);
		}

		public void FromCopy(Comment obj)
		{
			this.compn_sn = obj.compn_sn;
			this.contents = obj.contents;
		}
	}
}

File: src/SOP8/Egovframework/Base/base-model/src/main/java/egovframework/base/model/sop/component/Comment.java
package egovframework.base.model.sop.component;

import lombok.*;
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Comment {
	public enum Fields { compn_sn, contents }

	private int compn_sn;

	@NonNull
	@Builder.Default
	private String contents = "";

	public static final String autoIncreaseField = null;
	public static final String tableName = "so_compn_cm";
	public static final String[] primaryKeys = { "compn_sn" };

}
